### PR TITLE
replace @imports

### PR DESCRIPTION
**Pull request by webbird**
See issue #59

### CODE REVIEW

File: src/Munee/Asset/Type/Css.php [hunk 1]
```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
```
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: Please use `protected`, also put a space after the comma in the params.  This goes for any params and comma delimited lists.

File: src/Munee/Asset/Type/Css.php [hunk 2]
```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
...
+    {
+        $dir   = dirname($origFile);
+        // matches any type of import rule
+        preg_match_all('~@import\s*(?:url)?(?:\(?\'?\"?)?([^\'\"\)\(]*)(?:\'?\"?)?\)?\s?([^;]*);~im',$content,$imports,PREG_SET_ORDER);
+        if(count($imports))
```
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: I do not believe you need this if statement.  The `preg_match_all` will create the `$imports` as a blank array if it doesn't match anything.  Which means this loop won't run anyway.

File: src/Munee/Asset/Type/Css.php [hunk 3]
```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
...
+        {
+            foreach($imports as $i => $item)
+            {
```
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: Please put all curly braces on the same line (except for method definitions).

File: src/Munee/Asset/Type/Css.php (hunk 1 above)
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: Please also add a docblock.  And also, why did you need this as a static class?  It's already instantiated and this method can't be used outside of this class.

File: src/Munee/Asset/Type/Css.php [hunk 4]
```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
...
+                        // replace imports in current file
+                        $string = self::parseImports($string,$file);
+                        // replace urls
+                        if ($newDir !== $dir) {
+                            $tmp = $dir.'/';
+                            if (substr($newDir, 0, strlen($tmp)) === $tmp) {
+                                $string = preg_replace('#\burl\(["\']?(?=[.\w])(?!\w+:)#', '$0' . substr($newDir, strlen($tmp)) . '/', $string);
+                            }
+                        }
+                    if($media && $media != '')
```
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: Easiest to just use `if (! empty($media)) {`

**Comment by meenie** [comment]
This is good work so far :).  What also needs to happen is updating the documentation as well as writing tests.

File: src/Munee/Asset/Type/Css.php (hunk 1 above)
Comment (webbird) [inline src/Munee/Asset/Type/Css.php]: Sorry, that's a habit. In BlackCat CMS, especially in helper classes, all methods are declared as static but can be called in object context, too.

File: src/Munee/Asset/Type/Css.php (hunk 2 above)
Comment (webbird) [inline src/Munee/Asset/Type/Css.php]: That's also a habit. I always check double. In fact, normally I would do it like so:

    if(is_array($imports) && count($imports)) {

File: src/Munee/Asset/Type/Css.php (hunk 4 above)
Comment (webbird) [inline src/Munee/Asset/Type/Css.php]: empty() is not the same, as a content of 0 would be empty, too. ;) I tend to be very strict.

**Comment by meenie** [comment]
Thanks a ton for your suggestion and help on this!  I'll merge this now and hopefully write some tests around it this weekend.
